libraries/classes/Database/Qbe.php

```diff
@@ -250,7 +250,7 @@ class Qbe
      * @param DatabaseInterface $dbi             DatabaseInterface object
      * @param string            $dbname          Database name
      * @param array             $savedSearchList List of saved searches
-     * @param SavedSearches     $currentSearch   Current search id
+     * @param ?SavedSearches    $currentSearch   Current search id
```

Comment:
```suggestion
     * @param SavedSearches|null    $currentSearch   Current search id
```

For now we stick to the phpdoc standard :)